cockatrice/src/tab_game.cpp

```diff
@@ -309,6 +318,7 @@ TabGame::TabGame(TabSupervisor *_tabSupervisor, GameReplay *_replay)
     sayLabel(0),
     sayEdit(0)
 {
+    MainWindow = new QMainWindow;
```

Comment:
I believe the life of the object only lasts the time in which the tab is created and will be destroyed once the tab is removed.  Correct me if I am wrong, but shouldn't on destruct of the tab the object be removed?